internal/pkg/agent/application/monitoring/stats.go

```diff
@@ -12,6 +12,39 @@ import (
 	"github.com/elastic/elastic-agent-libs/monitoring"
 )
 
+const formValueKey = "failon"
+
+type LivenessFailConfig struct {
+	Degraded bool `yaml:"degraded" config:"degraded"`
+	Failed   bool `yaml:"failed" config:"failed"`
+}
+
+// process the form values we get via HTTP
+func handleFormValues(req *http.Request) (LivenessFailConfig, error) {
+	err := req.ParseForm()
+	if err != nil {
+		return LivenessFailConfig{}, fmt.Errorf("Error parsing form: %w", err)
+	}
+
+	defaultUserCfg := LivenessFailConfig{Degraded: false, Failed: true}
```

Comment:
Can the default be to rely only the on the coordinator check? That is, default `Failed: false`?

With the current implementation, I don't see how you could turn off the failed check if it turned out to be problematic. You'd have to stop using the liveness probe.

There are inputs that will fail because of invalid configurations, which isn't correctable via restarting, and also doesn't indicate the agent itself isn't "alive".